Authentication/Login/View/LoginController.swift

```diff
@@ -12,13 +12,19 @@ import Rex
 import enum Result.NoError
 
 
-/**
-    Log In View Controller that takes care of managing the login, from
+/*
+    Login View Controller that takes care of managing the login, from
     validating email and password fields, to binding a login view to the
-    view model and informing the log in controller delegate when certain
+    view model and informing the login controller delegate when certain
     events occur for it to act upon them.
-    If there are more than one errors in a field, the controller presents
-    only the first one in the errors label.
+    If there are more than one validation error in a field, the controller
+    presents only the first one in the errors label.
+ 
+    If wanting to use the default LoginController with some customization,
+    you should not override the `createLoginController` method of the Bootstrapper,
```

Comment:
Yeah that is true. My main concern is not being "coupled" with other parts of the library in a method or class documentation because, most probably, if something on how this class is used changes (not necessarily this class' API) then this docs would get out of date. Maybe the best place for high level usage documentation is the README, a tutorial or the Wiki.